name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/KeytradeBankPDFExtractor.java

```diff
@@ -149,16 +149,18 @@ private void addDividendeTransaction()
 
                 // Guthaben Kupons Ex-Kupon 29/06/2020
                 // CREDIT COUPONS Ex-coupon 01/02/2021
-                .section("date")
-                .match("^.* (Ex-Kupon|Ex-coupon|Kupons)(.*)? (?<date>[\\d]{2}\\/[\\d]{2}\\/[\\d]{4})$")
-                .assign((t, v) -> t.setDateTime(asDate(v.get("date").replaceAll("/", "."))))
+                .section("exDate")
+                .match("^.* (Ex-Kupon|Ex-coupon|Kupons)(.*)? (?<exDate>\\d{2}/\\d{2}/\\d{4})$")
```

Comment:
`Ex-Kupon  === Ex-coupon` but not for `Kupons` only, i think... :thinking:

Reply:
you could be right with that. 

Dividende 3 from https://divvydiary.com/de/symbols/US6516391066 lists 2016/12/10 as ex-day but the file for the tests lists `Guthaben Kupons Record date 08/12/2016`.

I changed this to
`.section("exDate").optional()
                .match("^.* (Ex-Kupon|Ex-coupon)(.*)? (?<exDate>\\d{2}/\\d{2}/\\d{4})$")
                .assign((t, v) -> t.setExDate(asDate(v.get("exDate").replaceAll("/", ".")).toLocalDate()))`